packages/core-components/api-report.md

```diff
@@ -647,7 +647,9 @@ export type LogViewerClassKey =
 
 // @public
 export interface LogViewerProps {
-  className?: string;
+  classes?: {
+    root?: string;
```

Comment:
was expecting `{ [key: LogViewerClassKey]?: string }` here, so that all of the styling flows through as expected

Reply:
Wanted to intentionally limit it for now. Local styling of the internals in plugins is likely not a good idea and it should probably be done with theme overrides instead